tests/ES.FX.Ignite.OpenTelemetry.Exporter.Seq.Tests/FunctionalTests.cs

```diff
@@ -0,0 +1,55 @@
+﻿using ES.FX.Shared.Seq.Tests.Fixtures;
+using ES.FX.Ignite.AspNetCore.HealthChecks.UI.Hosting;
+using ES.FX.Ignite.OpenTelemetry.Exporter.Seq.Hosting;
+using ES.FX.Ignite.Serilog.Hosting;
+using ES.FX.Ignite.Hosting;
+using Microsoft.AspNetCore.Builder;
+using Seq.Api;
+
+namespace ES.FX.Ignite.OpenTelemetry.Exporter.Seq.Tests
+{
+
+    public class FunctionalTests(SeqContainerFixture seqFixture)
```

Comment:
Convert to SUT. Call endpoint with client and check exported logs

Reply:
should be done, ty